src/nav/telemetry-data-platform.yml

```diff
@@ -120,7 +120,7 @@ pages:
             path: /docs/apis/intro-apis
             pages:
               - title: New Relic APIs
-                path: /docs/apis/get-started/intro-apis/introduction-new-relic-apis
+                path: /docs/apis/intro-apis/introduction-new-relic-apis
```

Comment:
I think you may also need to change this path in the [landing page](https://docs.newrelic.com/). I still get a 404 error.

Reply:
ok thanks